Makefile

```diff
@@ -235,7 +235,7 @@ ui-build:
 
 ## Build only Meshery UI on your local machine.
 ui-meshery-build:
-	cd ui; npm run $(UI_BUILD_SCRIPT) && npm run export; cd ..
+	cd ui; npm run build && npm run export; cd ..
```

Comment:
Why this change?

Reply:
For the purpose of testing Cypress

Reply:
I dont think this is affecting cypress at all. Now a new problem arises, that will stop us from building meshery with any node version later than 16